### ISSUE TITLE
Compile errors using maven

### ISSUE THREAD

**Issue by delta-bravo** [bane73/firebase4j#6]
mvn clean install causes compilation errors related to dependencies.
When dependencies versions are changed in pom.xml the project builds OK.
Updated pom.xml (resulting in correct build):

` <dependencies>

```
<dependency>
    <groupId>org.apache.httpcomponents</groupId>
    <artifactId>httpclient</artifactId>
    <version>4.3.6</version>    
</dependency>

<dependency>
    <groupId>log4j</groupId>
    <artifactId>log4j</artifactId>
    <version>1.2.16</version>   
</dependency>

<dependency>
    <groupId>commons-logging</groupId>
    <artifactId>commons-logging</artifactId>
    <version>1.1.3</version>    
</dependency>

<dependency>
    <groupId>org.codehaus.jackson</groupId>
    <artifactId>jackson-mapper-asl</artifactId>
    <version>1.9.5</version>    
</dependency>
```

  </dependencies>`

**Comment by bane73** [comment]
@delta-bravo thank you for bringing this to my attention.  I can make the changes, no sweat, but if you'd prefer you can issue a pull-request and I'll accept it.  Just let me know how you'd prefer to proceed, either way is fine with me.

**Comment by delta-bravo** [comment]
Sure, I will this week

**Comment by bane73** [comment]
Corrected by[Commit #7](https://github.com/bane73/firebase4j/pull/7)
